README.md

```diff
@@ -16,12 +20,123 @@ For npm users:
+        </FormLayoutRow>
+      </FormCard>
+    </Block>
+  );
+}
+```
+
+## Components
+
+### [FormCard](https://github.com/istreamlabs/pebble-formik/blob/main/src/FormCard.tsx)
+
+FormCard was designed to be an all-in-one form wrapper with a submit and reset button in the footer. It includes a prompt message if the user tries to navigate away from.
+
+### [FormGroup](https://github.com/istreamlabs/pebble-formik/blob/main/src/FormGroup.tsx)
+
+FormGroup creates a visual grouping of your form components and has options for a title: `string | node`, and a subtitle: `string`. If you pass in `true` for the `collapsible` prop, the group turns into a [collapsible panel](https://pebble.istreamplanet.net/#/Components/CollapsiblePanel).
+
+### Form Components
+
+Form components are just Pebble components in a Formik wrapper, where Pebble has `value` these get their values through the `name` prop. Match the key of the value you want to the `name`.
+
+*Example*
+```shell
+// incoming data
+{
+  description: 'Cat is on a diet',
+}
+
+<FieldText name='description'>
+```
+
+* [FieldCheckbox](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldCheckbox.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldCheckbox)
+* [FieldCheckboxInAGroup](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldCheckboxInAGroup.tsx)
+* [FieldDateTime](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldDateTime.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldDateTime)
+* [FieldNumber](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldNumber.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldNumber)
+* [FieldRadioGroup](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldRadioGroup.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldRadioGroup)
+* [FieldSelect](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldSelect.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldSelect)
+* [FieldText](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldText.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldText)
+* [FieldTextDebounce](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldTextDebounce.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldTextDebounce)
+* [FieldToggle](https://github.com/istreamlabs/pebble-formik/blob/main/src/FieldToggle.tsx) / [PebbleComponent](https://pebble.istreamplanet.net/#/Components/FieldToggle)
+
+### Error Components
+
+Formats error messages in a standard, easy-to-read manner.
+
+* [ErrorMessage](https://github.com/istreamlabs/pebble-formik/blob/main/src/ErrorMessage.tsx) : Has a `small` prop that fits on a single form row.
+* [ErrorSummary](https://github.com/istreamlabs/pebble-formik/blob/main/src/ErrorSummary.tsx) : Is built in to the bottom of FormCard.
+
+## How to Release
```

Comment:
should be in development file similar to https://github.com/istreamlabs/pebble/blob/main/DEVELOPMENT.md

Reply:
added https://github.com/istreamlabs/pebble-formik/blob/ux-949/DEVELOPMENT.md